nucypher/blockchain/eth/actors.py

```diff
@@ -217,8 +221,8 @@ def __repr__(self):
 
     @validate_checksum_address
     def recruit_sidekick(self, sidekick_address: str, sidekick_password: str):
-        self.sidekick_power = TransactingPower(account=sidekick_address, password=sidekick_password, cache=True)
-        if self.sidekick_power.device:
+        self.sidekick_power = TransactingPower(checksum_address=sidekick_address, password=sidekick_password, cache=True)
+        if self.sidekick_power.is_device:
```

Comment:
This will fail currently fail but it's not tested. See my comment in the init of TransactingPower.

Reply:
:heavy_check_mark: